src/tests/unit/electron/platform/android/scan-results.test.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't think this file should be .tsx. Those are usual react/ui related files.

Reply:
Renamed to scan-results.test.ts